src/lib/shared/components/ui/Jumbotron/Jumbotron.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box } from "@mui/material";
+import { SxProps, Theme } from '@mui/material/styles';
+import { Button, Paragraph } from '@/lib/shared/components/ui';
+
+import { styled } from '@mui/material/styles';
+import { H1 } from '@/lib/shared/components/ui';
+
+export const TEST_IDS = {
+    CONTAINER: 'container',
+} as const;
+
+export interface JumbotronProps {
+    headerText: string;
+    subHeaderText?: string;
+    backgroundImageUrl?: string;
+    callToAction?: {
+        label: string;
+        onClick?: () => void;
+    }
+    sx?: SxProps<Theme>
+}
+
+export function Jumbotron({ sx, headerText, subHeaderText, backgroundImageUrl, callToAction }: JumbotronProps) {
+    return (
+        <Container data-testid={TEST_IDS.CONTAINER} backgroundImageUrl={backgroundImageUrl} sx={sx}>
+            <ContentContainer>
+                <Header>{headerText}</Header>
+                {subHeaderText ?? <SubHeader>{subHeaderText}</SubHeader>}
+                {callToAction && <Button aria-label="button" onClick={callToAction.onClick}>{callToAction.label}</Button>}
+            </ContentContainer>
+        </Container>
+    );
+}
+
+const Header = styled(H1)(({ theme }) => ({
+    ...theme.typography.h4,
+}));
+
+const SubHeader = styled(Paragraph)(({ theme }) => ({
+    ...theme.typography.body1,
```

Comment:
Can we add a little margin to the bottom of the subheader? The spacing looked kind of tight to the button in the storybook picture you posted. Im not seeing any uses of a subheader and a button in the mock ups, but I think it would look a little nicer with maybe 12 or 16 px of margin. If this ends up looking stupid, lets reach out to design about it just to make sure were all on the same page.